lib/zlib.js

```diff
@@ -56,7 +56,9 @@ const codes = {
 };
 
 const ckeys = Object.keys(codes);
-for (var ck = 0; ck < ckeys.length; ck++) {
+  
+const ckeyslen = ckeys.length ;
```

Comment:
Caching the array length isn't really necessary anymore. V8 should take care of it.